lib/ui/widgets/menu/menu_item.dart

```diff
@@ -5,51 +5,58 @@ class MenuItem extends StatelessWidget {
   final IconData? leadingIcon;
   final String text;
   final VoidCallback? onPressed;
+  final String? secondaryText;
   final bool isVisible;
 
   const MenuItem({
     super.key,
     this.leadingIcon,
     required this.text,
     this.onPressed,
+    this.secondaryText,
     this.isVisible = true,
   });
 
   @override
   Widget build(BuildContext context) {
-    return isVisible
-        ? InkWell(
-            onTap: onPressed,
-            child: Container(
-              height: 56,
-              decoration: BoxDecoration(
-                color: ZOColors.cardBackground,
-                borderRadius: BorderRadius.circular(8),
-              ),
-              child: Padding(
-                padding: const EdgeInsets.symmetric(
-                  horizontal: WidgetStyle.padding,
-                ),
-                child: Row(
-                  children: [
-                    if (leadingIcon != null) ...[
-                      Icon(leadingIcon),
-                      const SizedBox(width: GapSize.xs),
-                    ],
-                    const SizedBox(width: GapSize.xs),
-                    Text(text, style: const TextStyle(fontSize: FontSize.s)),
-                    const Spacer(),
-                    onPressed != null
+    if (isVisible) {
+      return InkWell(
```

Comment:
Pokud chapu spravne, tak `InkWell` by mel prave delat ripple, ale nejak nefunguje 🤔 
Budeme chtit to nekdy opravit?

Reply:
Dotaz prenesen do Slacku